pages/linux/progress.md

```diff
@@ -0,0 +1,16 @@
+# progress
+
+> Display/Monitor the progress of running coreutils.
+> More information: <https://github.com/Xfennec/progress>.
+
+- Show the progress of running coreutils:
+
+`progress`
+
+- Show the progress of running coreutils with minimized output:
```

Comment:
We usually try to match the option with the description text. Here `-q` refers to quiet. So we usually try to mention the word quiet so that is easy to understand what the command does.

I would suggest something like ".. coreutils in quiet mode"

Reply:
Thanks